app/views/admin/users/edit.html.erb

```diff
@@ -1,37 +1,50 @@
 <%= form_with(model: @user, url: admin_user_path(@user), method: 'patch' , local: true) do |f| %>
+  <% if @user.errors.any? %>
```

Comment:
add a partial for error messages in shared folder and use that partial in all views.